apps/studio/components/interfaces/Settings/Logs/LogSelection.tsx

```diff
@@ -34,15 +36,43 @@ const LogSelection = ({
   onClose,
   queryType,
   params = {},
+  collectionName,
 }: LogSelectionProps) => {
   const { logData: fullLog, isLoading } = useSingleLog(
     projectRef,
     queryType,
     params,
     partialLog?.id
   )
+  const [sql, setSql] = useState('')
+
+  const warehouseLogQuery = useWarehouseQueryQuery({
```

Comment:
minor: just for consistency

`const { data: warehouseLogData, refetch: refetchWarehouseData, ... } = useWarehouseQueryQuery(...)`